rise/lib/resolvers/categories.ts

```diff
@@ -1,21 +1,40 @@
 import { gql } from 'apollo-server-express'
-import { getCategories } from '../services/categories'
+import { getCategories, getCategoryById } from '../services/categories'
+import { getServicesByCategory } from '../services/riseServices'
 
 export const typeDefs = gql`
+  type Service {
+    id: String!
+    name: String!
+    link: String!
+    categories: [Category!]!
+  }
+
   type Category {
     id: String!
     label: String!
     introduction: String!
     information: String!
+    services: [Service]
   }
 
   extend type Query {
     categories: [Category!]!
+    categoryAndRelated(id: String!): Category!
   }
 `
 
 export const resolvers = {
   Query: {
-    categories: getCategories,
+    categories: () => getCategories,
+    categoryAndRelated: async (_ : any, { id }: { id: string }) => {
```

Comment:
Can we change the any's? If we'd use the `QueryResolvers['categoryAndRelated']` we could get type-inference from that for the `parent` and `args`